src/main/kotlin/com/sistemas/ferramentaquiz/api/controller/QuizController.kt

```diff
@@ -0,0 +1,21 @@
+package com.sistemas.ferramentaquiz.api.controller
+
+import com.sistemas.ferramentaquiz.api.request.CreateQuizRequest
+import com.sistemas.ferramentaquiz.service.QuizService
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/quiz")
+class QuizController(
+    val service: QuizService
+) {
+
+
+    @PostMapping("")
```

Comment:
```suggestion
    @PostMapping
```

Não precisa colocar parâmetro se não for colocar uma URI especifica